justfile

```diff
@@ -2,9 +2,10 @@ default:
     just --summary --unsorted
 
 test $RUST_BACKTRACE="1" *args="":
-    cargo build --release -p pathfinder --bin pathfinder -F p2p
-    cargo nextest run --no-fail-fast --all-targets --features p2p,consensus-integration-tests --workspace --locked \
-    -E 'not test(/^p2p_network::sync_handlers::tests::prop/)' \
+    cargo build --release -p pathfinder --bin pathfinder -F p2p,consensus-integration-tests
+    PATHFINDER_TEST_ENABLE_PORT_MARKER_FILES=1 cargo nextest run --test consensus -p pathfinder --features p2p,consensus-integration-tests --locked
+    cargo nextest run --no-fail-fast --all-targets --features p2p --workspace --locked \
+    -E 'not (test(/^p2p_network::sync_handlers::tests::prop/) | test(/^test::consensus_3_nodes/))' \
```

Comment:
I think we need the same for `test-all-features` as well?

Reply:
https://github.com/eqlabs/pathfinder/pull/3086/commits/9db0f0dbeb5323f442f6ae702ff345ae076376d6